app/javascript/components/Results/Results.tsx

```diff
@@ -1,27 +1,31 @@
+/* eslint-disable complexity */
+
 import React from 'react';
 import { GridContainer, Grid } from '@trussworks/react-uswds';
 
 import { Pagination } from './../Pagination/Pagination';
 
 import './Results.css';
+
 interface ResultsProps {
-  results: {
+  query?: string
+  results?: {
     title: string,
     url: string,
-    thumbnail: {
+    thumbnail?: {
       url: string
     },
     description: string,
-    updatedDate: string,
-    publishedDate: string,
-    thumbnailUrl: string
-  }[];
+    updatedDate: string | null,
```

Comment:
There are several new `null` types introduced in PR. Some of them should've been added in previous PRs, and some are needed due to how Bing pagination behaves.